metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceImpl.java

```diff
@@ -313,28 +317,39 @@ public void notifyOfTableUpdate(final MetacatUpdateTablePostEvent event) {
             Metrics.CounterSNSNotificationTableUpdate.getMetricName());
     }
 
-    private UpdateTableMessage createUpdateTableMessage(
+    private UpdateOrRenameTableMessageBase createUpdateorRenameTableMessage(
```

Comment:
I feel it's better to have two methods createUpdateTableMessage() and createRenameTableMessage(), each of them will call this createUpdateorRenameTableMessage() with the corresponding message type.

Reply:
Thanks Zhen. Normally I'd prefer the same, but in this case, the difference is just a statement (given the somewhat large method signature). I figured this should suffice as a middle-ground.